honeybadgermpc/hyper_invertible.py

```diff
@@ -0,0 +1,104 @@
+import asyncio
+import logging
+from honeybadgermpc.exceptions import HoneyBadgerMPCError
+from honeybadgermpc.polynomial import EvalPoint, polynomials_over
+from honeybadgermpc.reed_solomon import Algorithm, EncoderFactory, DecoderFactory
+from honeybadgermpc.batch_reconstruction import subscribe_recv, wrap_send
+
+
+class HyperInvMessageType(object):
+    SUCCESS = "S"
+    ABORT = "A"
+
+
+async def _recv_loop(n, recv, s=0):
+    results = [None]*n
+    for _ in range(n):
+        sender_id, value = await recv()
+        results[sender_id-s] = value
+    return results
+
+
+async def generate_double_shares(n, t, my_id, _send, _recv, field):
+    poly = polynomials_over(field)
+    eval_point, algorithm = EvalPoint(field, n, use_fft=True), Algorithm.FFT
+    big_t = n-2*t-1  # This is same as `T` in the HyperMPC paper.
```

Comment:
```suggestion
big_t = n - 1 - (t<<1) 
```
Even if you stick with `2*t`, I think this expression is hard to read without spacing

Reply:
Added more spaces. `2*t` i s more readable than `t<<1`. This operation is being done only once and using bit shift will hardly have any impact, in fact, the interpreter might even execute it like a shift.